botany_importer.py

```diff
@@ -88,12 +89,16 @@ def process_barcode(self, barcode, filepath_list):
         filepath_list = self.clean_duplicate_image_barcodes(filepath_list)
         filepath_list = self.remove_imagedb_imported_filenames_from_list(filepath_list)
 
```

Comment:
alternate agent_ID for image db depending on if active or passive import, for easier puring